core/cmd/app.go

```diff
@@ -92,8 +92,13 @@ func NewApp(client *Client) *cli.App {
 					Action: client.ReplayFromBlock,
 					Flags: []cli.Flag{
 						cli.IntFlag{
-							Name:  "block-number",
-							Usage: "Block number to replay from",
+							Name:     "block-number",
+							Usage:    "Block number to replay from",
+							Required: true,
+						},
+						cli.BoolFlag{
+							Name:  "force-broadcast",
+							Usage: "Whether to broadcast all logs from the provided block number and beyond, ignoring whether they were previously consumed",
```

Comment:
Seems like this could be tightened up a bit :shrug: 
```suggestion
							Usage: "Whether to force broadcasting logs which were already consumed and that would normally be skipped",
```
Also, would just `-force` for the name be too short?

Reply:
Maybe rather than "that would normally be skipped" we can say "that would otherwise be skipped" (i.e "otherwise" referring to the fact that if the flag is _not_ provided, we'd skip broadcasting already consumed logs).

`-force` sounds good to me.

Reply:
SGTM, done